frontend/src/features/admin-form/common/components/CollaboratorModal/CollaboratorModal.tsx

```diff
@@ -13,11 +13,13 @@ import { CollaboratorWizardProvider } from './CollaboratorWizardContext'
 interface CollaboratorModalProps {
   isOpen: boolean
   onClose: () => void
+  formId: string | undefined
```

Comment:
should we make this non-optional? or is there a reason why this can be optional? Parent of this component should make sure there is a formId before instantiating this component imho.